tests/test_translating_parser.py

```diff
@@ -0,0 +1,183 @@
+from os.path import join
+
+from prance import _TranslatingParser
+
+
+class SpecificationTester:
+    """
+    Helper class that tests correct reference translation. Loads a specification file from tests/specs, parses it with
+    the TranslatingParser and provides assertion methods:
+    * assert_schemas to check what’s in /components/schemas.
+    * assert_path_ref to check the $ref of the only operation response in the specification.
+    * assert_schema_ref to check the $ref of a schema in /components/schemas.
+
+    Typical usage:
+    tester = SpecificationTester("my_spec_file")
+    tester.assert_path_ref("some_other_refd_file.spec.yaml_SomeObject")
+    tester.assert_schemas(
+        {"some_other_refd_file.spec.yaml_SomeObject", "some_other_refd_file.spec.yaml_SomeOtherObject"}
+    )
+    tester.assert_schema_ref(
+        "some_other_refd_file.spec.yaml_SomeObject", "some_other_refd_file.spec.yaml_SomeOtherObject",
+    )
+    """
+    @staticmethod
+    def _parse_spec(name):
```

Comment:
lets turn this into a namd ctor

so we can do `SpecificationTester.from_example(name)`